.yarnrc.yml

```
@@ -0,0 +1,2 @@
+enableImmutableInstalls: false
```

```suggestion
enableImmutableInstalls: true
```

While having it set to `true` might be annoying at times when `yarn` resolves to a different typescript patch version, it generally makes workflows more secure and ensures the lockfile does not get outdated.

Having the immutable installs enabled is our recommendation for all extensions and the template dropped this line altogehter (https://github.com/jupyterlab/extension-template/pull/39) but I would suggest having it explicitly on.